net/private_net.go

```diff
@@ -44,7 +46,9 @@ func IsPrivateAddress(ip net.IP) bool {
 			return true
 		}
 	}
-	return false
+
+	ipt, _ := iptool.New()
+	return ipt.IsPrivate(&net.IPAddr{IP: ip})
```

Comment:
This is concise and works, but I suggest we instead add to the existing private ranges defined in this file (in the `init` function).  This way we can try to get the fix merged upstream - which we should definitely do IMO.  We also avoid adding redundant logic this way.

Reply:
To be clear, I'm saying that we should just take the private ranges [defined in `iptool`](https://github.com/getlantern/iptool/blob/c00e863b2696eeb5fe4fb20b33c7918d69bbb9ba/iptool.go#L15-L43) and add them to the ranges defined in the `init` function here.

Reply:
Sorry @hwh33 this had fallen off my radar. Going to implement this feedback.